docs/docsite/rst/getting_started/index.rst

```diff
@@ -1,99 +1,94 @@
 
-Inventory
-   A list of managed nodes that are logically organized.
-   You create an inventory on the control node to describe host deployments to Ansible.
+.. code-block:: shell
 
-.. image:: ../images/ansible_basic.svg
-   :width: 400px
-   :align: center
-   :height: 200px
-   :alt: Basic components of an Ansible environment include a control node, an inventory of managed nodes, and a module copied to each managed node.
+   $ touch inventory
 
-Ready to start using Ansible?
-Complete the following steps to get up and running:
+You can now add the necessary hosts and groups to the ``inventory`` file using your preferred text editor. Make sure to follow the correct INI file format.
 
-#. Install Ansible. Visit the :ref:`installation guide<installation_guide>` for complete details.
-   
-   .. code-block:: bash 
-      
-      python3 -m pip install --user ansible
+Step 3: Organize Playbooks
+--------------------------
+Now, let's create a directory to store your Ansible playbooks and place them adjacent to the inventory file. This will keep all your playbooks in one place, making it easier to manage and version control.
 
-#. Create an inventory by adding the IP address or fully qualified domain name (FQDN) of one or more remote systems to ``/etc/ansible/hosts``.
-   The following example adds the IP addresses of three virtual machines in KVM:
+.. code-block:: shell
 
-   .. code-block:: ini
+   $ mkdir playbooks
 
-      [myvirtualmachines]
-      192.0.2.50
-      192.0.2.51
-      192.0.2.52
+You can now start adding your Ansible playbooks to the ``playbooks`` directory. For example, you could create a playbook named ``setup.yml``:
 
-#. Verify the hosts in your inventory.
-   
-   .. code-block:: bash 
+.. code-block:: shell
 
-      ansible all --list-hosts
+   $ touch playbooks/setup.yml
 
-   .. code-block:: ansible-output
+Step 4: Manage Collections
+--------------------------
+To manage Ansible collections used in your project, create a directory called ``collections`` within the project directory.
 
-      hosts (1):
-        192.0.2.50
-        192.0.2.51
-        192.0.2.52
+.. code-block:: shell
 
-#. Set up SSH connections so Ansible can connect to the managed nodes.
+   $ mkdir collections
 
-   a. Add your public SSH key to the ``authorized_keys`` file on each remote system.
-   b. Test the SSH connections, for example:
+You can now add the necessary collections to a ``requirements.yml`` file inside the ``collections`` directory. This file will serve as a manifest for the collections your project depends on.
 
-   .. code-block:: bash
+.. code-block:: shell
 
-      ssh username@192.0.2.50
-    
-   If the username on the control node is different on the host, you need to pass the ``-u`` option with the ``ansible`` command.
+   $ touch collections/requirements.yml
 
-#. Ping the managed nodes.
+Inside the ``requirements.yml`` file, list the collections you need, specifying the name and version. For example:
```

Comment:
A requirements.yml is not needed for anyone just getting started. This should also be an additional topic saved for later.